model.py

```diff
@@ -872,8 +876,50 @@ def lookup(cls, _db, name):
             raise ValueError(_("Unknown audience: %(name)s", name=name))
         audience, is_new = get_one_or_create(_db, Audience, name=name)
         return audience
+
+class CollectionSummary(Base):
+    """A summary of a collection held by a library.
     
+    We only need to know the language of the collection and
+    approximately how big it is.
+    """
+    __tablename__ = 'collectionsummaries'
     
+    id = Column(Integer, primary_key=True)
+    library_id = Column(Integer, ForeignKey('libraries.id'), index=True)
+    language = Column(Unicode)
+    size = Column(Integer)
+
+    @classmethod
+    def set(cls, library, language, size):
+        """Create or update a CollectionSummary for the given
+        library and language.
+
+        :return: An up-to-date CollectionSummary, or None if the language
+         is not recognized.
```

Comment:
This isn't true, it still returns a CollectionSummary if the language isn't recognized.